server/routes/request.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+const { userRequests, createRequest, updateRequest } = require("../controllers/request");
+
+router.route("/new").post(createRequest)
+router.route("/update").put(updateRequest);
+router.route("/all/:user_id").get(userRequests);
```

Comment:
so for proper naming convention, i would suggest either /${user_id}/requests OR /requests and the user_id actually would get passed in through params rather than part of the route. either option would be good :)

Reply:
Hopefully I got this right, I tried to make it use the route '/${userId}/requests' in this commit,
- https://github.com/hatchways/team-igneous/pull/57/commits/5feb5f847e8c909a1dc2c010a54e9de6a89dee8a

Then I changed the APICalls, and Queries accordingly. I also added some other extra parameters to some Queries, as I updated the request-model as-well in #9 BE: Request Model.